nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-kubernetes-bundle/nifi-framework-kubernetes-state-provider/src/main/java/org/apache/nifi/kubernetes/state/provider/KubernetesConfigMapStateProvider.java

```diff
@@ -70,6 +78,10 @@ public class KubernetesConfigMapStateProvider extends AbstractConfigurableCompon
 
     private final AtomicBoolean enabled = new AtomicBoolean();
 
+    private String CONFIG_MAP_NAME_FORMAT;
+
+    private Pattern CONFIG_MAP_NAME_PATTERN;
```

Comment:
These should be changed reflect that they are member variables.
```suggestion
    private String configMapNameFormat;

    private Pattern configMapNamePattern;
```